quickstarts/maven/hello-world/src/main/java/org/eclipse/jkube/sample/helloworld/App.java

```diff
@@ -14,10 +14,47 @@
 package org.eclipse.jkube.sample.helloworld;
 
 /**
- * Hello world!
+ * @Author: Wayne Kirimi
  */
+import com.sun.net.httpserver.HttpServer;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpExchange;
+import java.net.InetSocketAddress;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.logging.Logger;
+
 public class App {
-    public static void main(String[] args) {
-        System.out.println("Hello World!");
+    private static final Logger log = Logger.getLogger(App.class.getSimpleName());
+    private static int PORT = 8081;
```

Comment:
```suggestion
    private static int PORT = 8080;
```
Exposing port via minikube as NodePort service seems to have some problems with this port number. IANA reads this port as `sunproxyadmin`